scripts/update_changelog.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+#
+# Run this script from the top level directory of the trillian repo e.g.
+# with scripts/update_changelog.sh.
+#
+# GOPATH must be set.
+
+set +e
+d=${GOPATH[0]}
+PATH=$PATH:${d}/bin
+
+# Get and build the correct branch that includes markdown output
```

Comment:
I *think* you could skip a bunch of these steps if you set the default branch for your GitHub fork to be `add_changelog_output` rather than `master`

Reply:
Wouldn't this be a bit fragile? Not that I plan to do much more work on that fork.

Reply:
Not sure it will be any more fragile than the current version:
 - the current variant will break if/when your PR is accepted and you delete the branch
 - the different-default variant will break if you change default branch to something without the fix, e.g. back to `master` before the PR is accepted.

I just like that the end result might be super-simple (if it works):
```bash
set +e
# TODO(Martin2112): replace with upstream repo if/when aktau/github-release#81 is merged
go install github.com/Martin2112/github-release
${GOPATH[0]}/bin/github-release info -r trillian -u google --markdown > CHANGELOG.md
```

Reply:
I got a scary warning from GitHub about "unintended consequences" but it seems to work.